docs/api-documentation/rpc.md

```diff
@@ -50,64 +68,85 @@ TBD
 
 `query(path: string, data: bytes)`: queries information in the state machine / database. Where `path` can be:
 
-* `account/<account_id>` - returns view of account information, e.g. `{"amount": 1000000, "nonce": 102, "account_id": "test.near"}`
+* `account/<account_id>` - returns view of account information.
 * `access_key/<account_id>` - returns all access keys for given account.
 * `access_key/<account_id>/<public_key>` - returns details about access key for given account with this public key. If there is no such access key, returns nothing.
 * `contract/<account_id>` - returns full state of the contract \(might be expensive if contract has large state\).
 * `call/<account_id>/<method name>` - calls `<method name>` in contract `<account_id>` as view function with `data` as parameters.
```

Comment:
also let's add validators (it's in staging now but is coming):
```
http post https://staging-rpc.nearprotocol.com/ jsonrpc=2.0 method=query id=dontcare 'params:=["validators", ""]'
HTTP/1.1 200 OK
Alt-Svc: clear
Via: 1.1 google
content-length: 366
content-type: application/json
date: Mon, 09 Sep 2019 19:12:35 GMT

{
    "id": "dontcare",
    "jsonrpc": "2.0",
    "result": {
        "current_proposals": [],
        "current_validators": [
            {
                "account_id": "near",
                "amount": "90604618001209940879281555",
                "public_key": "ed25519:DuZSg3DRUQDiR5Wvq5Viifaw2FXPimer2omyNBqUytua"
            }
        ],
        "next_validators": [
            {
                "account_id": "near",
                "amount": "94405848217684573203378772",
                "public_key": "ed25519:DuZSg3DRUQDiR5Wvq5Viifaw2FXPimer2omyNBqUytua"
            }
        ]
    }
}
```

Reply:
Done